llvm/test/CodeGen/SystemZ/xray-01.ll

```diff
@@ -0,0 +1,39 @@
+; RUN: llc < %s -mtriple=s390x-linux-gnu | FileCheck %s
```

Comment:
Perhaps just `xray.ll`

When we need other tests, they are likely suffixed, e.g. `xray-custom-log.ll`